Arrowgene.Ddon.Shared/Model/Quest/QuestBoardBaseId.cs

```diff
@@ -26,6 +26,7 @@ public enum QuestBoardBaseId : uint
 
         RathniteFoothills = 26, // All of Rathnite Foothills
         FelyanaWilderness = 27, // All of Felyana Wilderness
+        SpecialQuestBoard = 28, // In WDT, possibly elsewhere.
```

Comment:
I found there is a packet which says it refreshes the special quest board. Not sure if you saw that or not.

Reply:
S2C_QUEST_11_68_16_NTC is the packet which prints it in the quest log.